File: app/src/main/java/com/panda/ap_anime/anmi_details/details_repo/TralerResponse.java

package com.panda.ap_anime.anmi_details.details_repo;

import java.util.List;

import com.google.gson.annotations.SerializedName;
import qenawi.panda.a_predator.network_Handeler.CService_DBase;


public class TralerResponse extends CService_DBase {

    @SerializedName("id")
    private Long mId;
    @SerializedName("results")
    private List<Trailer_obj> mResults;

    public Long getId() {
        return mId;
    }

    public void setId(Long id) {
        mId = id;
    }

    public List<Trailer_obj> getResults() {
        return mResults;
    }

    public void setResults(List<Trailer_obj> results) {
        mResults = results;
    }

    @Override
    public boolean Is_Data_Good() {
        return this.mResults != null;
    }
}

File: app/src/main/java/com/panda/ap_anime/anmi_details/details_repo/TralerResponse.kt
package com.panda.ap_anime.anmi_details.details_repo

import com.google.gson.annotations.SerializedName
import qenawi.panda.a_predator.network_Handeler.CService_DBase


class TralerResponse : CService_DBase() {

    @SerializedName("id")
    var id: Long? = null
    @SerializedName("results")
    var results: List<Trailer_obj>? = null

    override fun Is_Data_Good(): Boolean {
        return this.results != null
    }
}
